keycloak/deploy/docker/docker-entrypoint.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+
+KEYCLOAK_USER=admin
+KEYCLOAK_PASSWORD=ash12345^&*()
+
+if [ $KEYCLOAK_USER ] && [ $KEYCLOAK_PASSWORD ]; then
```

Comment:
This must only be executed when a new environment is being created for the first time. This cannot be executed every time.